backend/game_history/views.py

```
@@ -0,0 +1,41 @@
+class FetchCompletedGamesView(APIView):
+    def post(self, request):
+        try:
+            user = request.user
+            fields_to_include = ["id", "white_player", "black_player", "game_winner"]
+
+            games_played = ChessGame.objects.filter(Q(white_player=user) | Q(black_player=user))
+            completed_games = games_played.filter(game_status="Ended")
+            serialized_completed_games = serialize_models_including_fields(completed_games, fields_to_include)
+
+            return Response(serialized_completed_games, status=status.HTTP_200_OK)
+        except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)